{{cookiecutter.plugin_name}}/setup.cfg

```diff
@@ -74,7 +74,7 @@ where = src
 
 [options.package_data]
 {{cookiecutter.plugin_name}} = 
-    napari.yaml
+    napari.yml
```

Comment:
we should decide what convention we're going to use. I've generally been going with `napari.yaml`.  YAML [prefers](https://yaml.org/faq.html) we use `.yaml` over `.yml`,  but we should just be consistent

Reply:
I'm ok with just deciding on `.yaml`

Reply:
>  auto-formatter for markdown?

I will do my best :) did that affect this pr?